### ISSUE TITLE
Linux support?

### ISSUE THREAD

**Issue by zohozer** [oliverschwendener/ueli#417]
Hi. I am a user of Ueli for MacOS, I do really enjoing using on may MacBookPro machine.

However, I do installed an Ubuntu 20.04 OS on my other machine, a MacMini and I do wanted to install Ueli also in this machine but only after reading the readme.md I do realized that Ueli it is running only on Windows and MacOS?

It is possible to port Ueli also to Linux? I will be more than happy to test any beta versions!

P.S: as this project seems to be an Electron app, hopefully will be easy to have a Linux version.

**Comment by oliverschwendener** [comment]
Duplicate of #94
